src/daemon/rpmostree-sysroot-upgrader.c

```diff
@@ -119,62 +119,52 @@ rpmostree_sysroot_upgrader_initable_init (GInitable        *initable,
                                           GCancellable     *cancellable,
                                           GError          **error)
 {
-  gboolean ret = FALSE;
   RpmOstreeSysrootUpgrader *self = (RpmOstreeSysrootUpgrader*)initable;
+
   OstreeDeployment *booted_deployment =
     ostree_sysroot_get_booted_deployment (self->sysroot);
-  const char *merge_deployment_csum = NULL;
-  gboolean is_layered;
-
   if (booted_deployment == NULL && self->osname == NULL)
-    {
-      g_set_error_literal (error, G_IO_ERROR, G_IO_ERROR_FAILED,
-                           "Not currently booted into an OSTree system and no OS specified");
-      goto out;
-    }
+    return glnx_throw (error, "Not currently booted into an OSTree system and no OS specified");
```

Comment:
80 char...max width for the code?  We'll have to negotiate this :handshake:    I find 100 chars more reasonable in C (where we have long function names).  But I also have a widescreen monitor.